src/mappers/collection/statistics/statistics.mapper.ts

```diff
@@ -0,0 +1,93 @@
+import { AbstractMapper } from '../../abstract.mapper';
+import {
+  Stat,
+  StatisticsDatashape,
+  TYPE,
+} from '../../../datashapes/statistics.datashape';
+import { ENTITIES } from '../../../entities/const';
+import { Protocol } from '../../../entities/protocol.entity';
+
+type Config = {
+  entity: string;
+};
+
+type Params = Record<string, never>;
+
+export class StatisticsMapper extends AbstractMapper<
+  Config,
+  Params,
+  any,
+  StatisticsDatashape
+> {
+  static getName(): string {
+    return 'Statistics';
+  }
+
+  static getDescription(): string {
+    return `Mapper that converts an array of specified values to statistics`;
+  }
+
+  static getType(): string {
+    return 'statistics';
+  }
+
+  public static getConfigSchema(): object {
+    return {
+      title: 'Config',
+      description: 'StatisticsMapper configuration',
+      type: 'object',
+      properties: {
+        entity: {
+          description: 'Entity to be processed, e.g. walletToken',
+          enum: ENTITIES,
+        },
+      },
+      required: ['entity'],
+    };
+  }
+
+  public static getParamsSchema(): object {
+    return {
+      title: 'Params',
+      description: 'StatisticsMapper params',
+      type: 'object',
+      properties: {
+        dappId: {
```

Comment:
wut, what does dappId has to do with a mapper?

Reply:
forgot it here